features/net/network-socket/nsapi_dns.cpp

```diff
@@ -261,20 +377,18 @@ static int nsapi_dns_query_multiple(NetworkStack *stack,
 }
 
 // convenience functions for other forms of queries
-extern "C" int nsapi_dns_query_multiple(nsapi_stack_t *stack,
-        nsapi_addr_t *addr, unsigned addr_count,
-        const char *host, nsapi_version_t version)
+extern "C" int nsapi_dns_query_multiple(nsapi_stack_t *stack, const char *host,
+        nsapi_addr_t *addr, unsigned addr_count, nsapi_version_t version)
 {
     NetworkStack *nstack = nsapi_create_stack(stack);
-    return nsapi_dns_query_multiple(nstack, addr, addr_count, host, version);
+    return nsapi_dns_query_multiple(nstack, host, addr, addr_count, version);
```

Comment:
The diff should now be split (with reason for reorder in commit message):
8c4ce18 nsapi - Reversed arguments to gethostbyname
9b0466f nsapi - Added access to gethostbyname and add_dns_server